crypto/argon2/argon2.c

```diff
@@ -0,0 +1,195 @@
+/*
+ * Argon2 reference source code package - reference C implementations
+ *
+ * Copyright 2015
+ * Daniel Dinu, Dmitry Khovratovich, Jean-Philippe Aumasson, and Samuel Neves
```

Comment:
ICLAs are needed from all original authors (irrespective of the reference implementation licence terms).

https://mta.openssl.org/pipermail/openssl-project/2019-July/001480.html

Reply:
I will look into it, thanks!